packages/utils/odsp-doclib-utils/src/odspRequest.ts

```diff
@@ -13,7 +13,7 @@ export async function getAsync(
     url: string,
     authRequestInfo: IOdspAuthRequestInfo,
 ): Promise<Response> {
-    return authRequest(authRequestInfo, async (config: RequestInit) => fetch(url, config) as Response);
+    return authRequest(authRequestInfo, async (config: RequestInit) => fetch(url, config) as Promise<Response>);
```

Comment:
can we just drop any casting here and let compiler figure out the type?
If not, does it point to a need to fix it in some other place?

Reply:
Yes we can drop it. Let me add lint disbale rule then.